src/librustc_target/abi/call/mod.rs

```diff
@@ -69,6 +69,7 @@ mod attr_impl {
 #[derive(Copy, Clone, PartialEq, Eq, Debug)]
 pub struct ArgAttributes {
     pub regular: ArgAttribute,
+    /// The dereferenceable size of the pointee.
```

Comment:
`dereferenceable` is LLVM jargon, if you want to document this I'd write something more like:
> The minimum size of the pointee, guaranteed to be valid for the duration of the whole call (corresponding to LLVM's `dereferenceable` and `dereferenceable_or_null` attributes).

Reply:
I'd use the same jargon in an eventual Rust spec, but sure.